packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -146,6 +149,7 @@ export const SubscriptionCreate = ({
               stripeFromParams,
             selectedPlan,
             customer,
+            profile,
```

Comment:
Same goes here as for `PaypalButton` in terms of passing down `checkoutType` instead of `profile`.

In the case of the `SubscriptionCreate` route (unlike for the `PaypalButton`), we know `checkoutType` is `'with-account'`.

Reply:
👍